doc/manual/parallel-computing.rst

```diff
@@ -444,6 +444,22 @@ preemptively. This means context switches only occur at well-defined
 points: in this case, when :func:`remotecall_fetch` is called.
 
 
+Channels
+--------
+Channels provide for a fast means of inter-task communication. A
+``Channel(T::Type, n::Int)`` is a shared queue of maximum length ``n``
+holding objects of type ``T``. Multiple readers can read off the channel
+via ``fetch`` and ``take!``. Multiple writers can add to the channel via
+``put!``. ``isready`` tests for the prescence of any object in
+the channel, while ``wait`` waits for an object to become available.
+
+A Channel can be used as an iterable object in a ``for`` loop, in which
+case the loop runs till the channel is open. The loop variable takes on
```

Comment:
It means empty queue and that `close` has not been called. The `for` loop will wait for more data even on a currently empty queue, as long as `close` has not been called. Similar to the `Task` iterator which runs till the task terminates.